emulators/qemu/Portfile

```diff
@@ -260,6 +254,14 @@ variant ssh description {Support remote block devices over SSH} {
     depends_lib-append      port:libssh
 }
 
+variant dbus description {Export the VM display through D-Bus} {
+    configure.args-replace  --disable-dbus-display --enable-dbus-display
+}
```

Comment:
I would expect this to need some additional library dependencies? Should we require the dbus daemon as well?

According to the meson.build in the qemu sources it needs at least few more client libraries: https://github.com/qemu/qemu/blob/823a3f11fb8f04c3c3cc0f95f968fef1bfc6534f/meson.build#L1458-L1465

Reply:
Hi @raimue,

I have updated the dbus variant to include the dependencies in from the meson file.  I did not add the dbus port as a required dependency, but I can do that, if you think it would be good it include it (FYI, The +dbus variant that I added seemed to build fine on my system without adding the dbus port as a required dependency).

Thanks,

-ranga